src/widgets/option-item/ui/option-item.module.scss

```diff
@@ -0,0 +1,22 @@
+.option-item {
+  display: flex;
+  flex-direction: column;
+  gap: 16px;
+
+  max-width: 380px;
+
+  color: $color-gray-600;
+
+  @include media-laptop {
+    max-width: 600px;
+    padding-top: 24px;
+  }
+
+  @include media-tablet {
+    max-width: 100%;
+  }
+}
+
+.option-description {
```

Comment:
Please, remove vertical `margins` as we discussed yesterday

Reply:
Done